common/src/main/java/com/wynntils/features/user/ItemHighlightFeature.java

```diff
@@ -87,10 +88,16 @@ public void onRenderSlot(SlotRenderEvent.Pre e) {
         if (!inventoryHighlightEnabled) return;
 
         ItemStack item = e.getSlot().getItem();
-        if (!(item instanceof HighlightedItem highlightedItem)) return;
```

Comment:
Given the similarities between this and the hotbar version, maybe you can extract this into a `getHighlightColor` method, which just returns `CustomColor.NONE` if it should not be highlighted (for whatever reason)?

Reply:
a helper method within the feature, you mean? i can do that, although it'd need a way to differentiate between hotbar and inventory too

Reply:
Yes, that what was I meant. 

I just thought it looked like the initial checks where identical between the two methods, up until the point of the actual rendering, so I thought that part could be extracted to a method which returns either null/Color.NONE if we should not render any highlight, and otherwise the color to use.

But it's not super important and it does not have to be done as part of this PR.